marker_api/src/ast/stmt.rs

```diff
@@ -1,24 +1,37 @@
-use crate::{context::with_cx, ffi::FfiOption, CtorBlocker};
+use std::fmt::Debug;
+use std::marker::PhantomData;
 
-use super::{
-    expr::ExprKind, item::ItemKind, pat::PatKind, ty::SynTyKind, LetStmtId, Span, SpanId, StmtId, StmtIdInner,
-};
+use crate::{ffi::FfiOption, private::Sealed};
+
+use super::{expr::ExprKind, item::ItemKind, pat::PatKind, ty::SynTyKind, Span, SpanId, StmtId};
+
+/// This trait combines methods, which all statements have in common.
+///
+/// This trait is only meant to be implemented inside this crate. The `Sealed`
+/// super trait prevents external implementations.
+pub trait StmtData<'ast>: Debug + Sealed {
```

Comment:
Should this and a bunch of other traits be a `prelude` of `marker_api`?

Reply:
Yes, should be :+1: